src/ImageSharp/Formats/Gif/Sections/GifXmpApplicationExtension.cs

```diff
@@ -67,7 +53,7 @@ public int WriteTo(Span<byte> buffer)
             int totalSize = this.ContentLength;
             if (buffer.Length < totalSize)
             {
-                throw new InsufficientMemoryException("Unable to write XMP metadata to GIF image");
+                ThrowInsufficientMemory();
```

Comment:
Yeah that’s an odd one. It could only ever be triggered if somehow the extension content length was incorrect which is impossible since we own the property. I’m gonna remove it.